packages/manager/apps/web/client/app/domain-operation/operation-table/operation-table.html

```diff
@@ -102,8 +102,7 @@
         data-compact
         data-disabled="!$row.canCancel && !$row.canRelaunch && !$row.canAccelerate && $row.function !== 'DomainIncomingTransfer'"
         data-placement="end"
-        data-oui-tooltip="{{:: 'domain_operations_no_action' | translate }}"
-        data-oui-tooltip-enable="{{!$row.canCancel && !$row.canRelaunch && !$row.canAccelerate}}"
+        data-oui-tooltip="{{ :: !$row.canCancel && !$row.canRelaunch && !$row.canAccelerate ? 'domain_operations_no_action' : '' | translate }}"
```

Comment:
Is this working without any common bracket ?

Reply:
data-oui-tooltip="{{ :: (!$row.canCancel && !$row.canRelaunch && !$row.canAccelerate ? 'domain_operations_no_action' : '') | translate }}"

Reply:
It's working

<img width="366" alt="image" src="https://github.com/user-attachments/assets/79aae070-844f-4601-b2be-5f96b2cdba43">